src/environments/environment.model.ts

```diff
@@ -138,6 +140,13 @@ export const ENVIRONMENT_DEFAULTS: Environment = {
     { lang: 'de_DE', currency: 'EUR', value: 'de', displayName: 'German', displayLong: 'German (Germany)' },
     { lang: 'fr_FR', currency: 'EUR', value: 'fr', displayName: 'French', displayLong: 'French (France)' },
   ],
+  // tslint:disable: use-camel-case-environment-properties
```

Comment:
instead of disabling this rule, adapt it 😉 https://github.com/intershop/intershop-pwa/blob/6a80bb74be4b07e49902bf6a1cd33cc4a783dd01/tslint-rules/src/useCamelCaseEnvironmentPropertiesRule.ts#L13